sdk/src/store.rs

```diff
@@ -418,70 +402,6 @@ impl Store {
         }
     }
 
-    /// Sign the claim and return signature.
```

Comment:
There was actually a reason this was in store.rs and not claim.rs.  claims don't sign themselves.  The are signed by outside actors.  They do not have the global knowledge that a store does.

Reply:
I've reverted all the reworking of signing